src/lib/registration/types.ts

```diff
@@ -92,7 +92,6 @@ export interface BeaconUse {
   windfarm: string;
   windfarmLocation: string;
   windfarmPeopleCount: string;
-  otherActivityDescription: string;
```

Comment:
This property was only used in mocks; no test/production code touched it, so I deleted it as unused. otherActivityText property remains and is used.